README.md

```diff
@@ -221,7 +221,7 @@ your local Python's `sqlite3` library ships with the `JSON1` extension enabled
 
 - Tests
   ```shell
-  nosetests
+  pynose 
```

Comment:
Preference for recommending running tests with nosetests vs pynose?

Reply:
No strong preference either way. "The fewer changes the better"?